app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,120 @@
+      </ul>
+      <p>
+        Themes you’ll cover for 16 to 18 year olds (key stage 5):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Fields, vectors and scalars</li>
+        <li class="tag">Mechanics and mechanical properties of matter</li>
+        <li class="tag">Electric circuits and waves</li>
+        <li class="tag">Matter, quantum and nuclear physics</li>
+      </ul>
+    <p>
+      <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">Read the national curriculum for more information about physics lessons</a>.
+    </p>
+
+    <p><a href="https://www.ogdentrust.com/teacher-support/training-to-be-a-teacher/">Find out more about becoming a physics teacher from the Ogden Trust</a>, a charitable trust which promotes the teaching and learning of physics. </p>
+  </section>
+   <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "It's inspiring getting 'wow, that's how it works' from pupils as they grasp a subject that they thought was hard.",
+    name: "Jane Doran, physics and robotics teacher, West Midlands",
+    background: "grey",
+    large: true
+  ) %>
+</section>
+
+ <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithOneImageComponent.new(
+      heading: "",
+      colour: "yellow",
+      image_path: "static/images/content/hero-images/0006.jpg",
+      image_alt: "Classroom full of pupils.",
+    ) %>
+  </section>
+  <section class="col col-space-m col-720">
+    <h2>Check your qualifications</h2>
+    <p>You'll need a bachelors degree before training to teach, but it does not have to be in physics.</p>
+<p>Physics teacher training courses have had successful applications from candidates with a range of degrees, including maths, geology, business studies and finance.</p>
+<p>To train to teach in primary and secondary schools in England, you’ll need:</p>
+<ul>
+<li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary)</li>
+<li>a bachelor’s degree in any subject</li>
+</ul>
+<p><a href="/non-uk-teachers/train-to-teach-in-england-as-an-international-student">Non-UK citizens can find out more about training to teach in England</a>.</p>
+<p>Your training provider may need you to brush up on your physics knowledge before you become a physics teacher. You can do a <a href="/how-to-apply-for-teacher-training/ske-explained-transcript">subject knowledge enhancement (SKE) course</a> before you start your teacher training to top up your subject knowledge.  </p>
+<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to teach</a>. </p>
+  </section>
+  <section class="col col-space-m col-720">
+<h2>Fund your teaching training</h2>
+To train to be a physics teacher, you'll need to do a teacher training course.
```

Comment:
@caitlin-reidy same as other page. This sentence doesn't sound quite right. To become a physics teacher, you'll need to do a teacher training course or the other way round You'll need to do a teacher training course to teach physics.